Cargo.toml

```diff
@@ -12,9 +12,6 @@ epoll = ">=4.0.1"
 lazy_static = "1.4.0"
 libc = "0.2.66"
 log = { version = "0.4.10", features = ["std"] }
-net_gen = { path = "net_gen" }
-net_util = { path = "net_util" }
-qcow = { path = "qcow" }
```

Comment:
Could you squash that into each commit responsible for moving the code?

Reply:
Done!